app/frontend/src/pages/chat/Chat.tsx

```diff
@@ -136,6 +137,7 @@ const Chat = () => {
     };
 
     const client = useLogin ? useMsal().instance : undefined;
+    const { loggedIn } = useLogin ? useContext(LoginContext) : { loggedIn: false };
```

Comment:
Would it make sense for LoginContext to have a function to return the { loggedin: false}? I notice you do that in a few files, and that it matches what an actual LoginContext has, so it might make sense for the dummy context to be in that file as well. Just an idea.

Reply:
Good idea!